tests/countries/test_anguilla.py

```diff
@@ -0,0 +1,335 @@
+            "2021-04-05",
+            "2022-04-18",
+            "2023-04-10",
+            "2024-04-01",
+            "2025-04-21",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, range(2001, 2050))
+
+    def test_easter_sunday(self):
+        name = "Easter Sunday"
+        dt = (
+            "2011-04-24",
+            "2012-04-08",
+            "2013-03-31",
+            "2014-04-20",
+            "2015-04-05",
+            "2016-03-27",
+            "2017-04-16",
+            "2018-04-01",
+            "2019-04-21",
+            "2020-04-12",
+            "2021-04-04",
+            "2022-04-17",
+            "2023-04-09",
+            "2024-03-31",
+            "2025-04-20",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, range(2001, 2050))
+
+    def test_labor_day(self):
+        name = "Labour Day"
+        self.assertHolidayName(name, (f"{year}-05-01" for year in range(2001, 2050)))
+        dt = ("2011-05-02", "2016-05-02", "2021-05-03", "2022-05-02")
+        self.assertHolidayName(f"{name} (observed)", dt)
+        self.assertNoNonObservedHoliday(dt)
+
+    def test_whit_monday(self):
+        name = "Whit Monday"
+        self.assertHolidayName(
+            name,
+            "2011-06-13",
+            "2012-05-28",
+            "2013-05-20",
+            "2014-06-09",
+            "2015-05-25",
+            "2016-05-16",
+            "2017-06-05",
+            "2018-05-21",
+            "2019-06-10",
+            "2020-06-01",
+            "2021-05-24",
+            "2022-06-06",
+            "2023-05-29",
+            "2024-05-20",
+            "2025-06-09",
+        )
+        self.assertHolidayName(name, range(2001, 2050))
+
+    def test_queen_birthday(self):
+        name = "Celebration of the Birthday of Her Majesty the Queen"
+        self.assertHolidayName(
+            name,
+            "2011-06-13",
+            "2012-06-11",
+            "2013-06-10",
+            "2014-06-09",
+            "2015-06-08",
+            "2016-06-13",
+            "2017-06-12",
+            "2018-06-11",
+            "2019-06-10",
+            "2020-06-08",
+            "2021-06-14",
+            "2022-06-03",
+        )
+        self.assertHolidayName(name, range(2001, 2023))
+        self.assertNoHolidayName(name, range(2023, 2050))
+
+    def test_king_birthday(self):
+        name = "Celebration of the Birthday of His Majesty the King"
+        self.assertHolidayName(
+            name,
+            "2023-06-19",
+            "2024-06-17",
+            "2025-06-16",
+        )
+        self.assertHolidayName(name, range(2023, 2050))
+        self.assertNoHolidayName(name, range(2001, 2023))
+
+    def test_anguilla_day(self):
+        name = "Anguilla Day"
+        self.assertHolidayName(name, (f"{year}-05-30" for year in range(2001, 2050)))
+        dt = ("2004-06-01", "2009-06-02", "2010-05-31", "2020-06-02", "2021-05-31")
```

Comment:
```suggestion
        dt = (
            "2004-06-01",
            "2009-06-02",
            "2010-05-31",
            "2015-06-01",
            "2020-06-02",
            "2021-05-31",
            "2039-05-31",
        )
```
2039 - Anguilla Day is in same day with Whit Monday.